python-threatexchange/threatexchange/exchanges/helpers.py

```diff
@@ -145,3 +148,91 @@ def get_for_signal_type(
                 if by_signal:
                     ret[collab.name] = by_signal
         return ret
+
+    def exists(self, collab: CollaborationConfigBase) -> bool:
+        raise NotImplementedError
+
+
+class DMBFetchedStateStore(fetch_state.FetchedStateStoreBase):
+    def __init__(
+        self,
+        api_cls: TSignalExchangeAPICls,
+    ) -> None:
```

Comment:
All of these methods end up taking a collaboration implicitly, and we could probably provide a wrapper that could cover the equivalent of get_for_signal_type.